### ISSUE TITLE
CKEditor not working for document level properties

### ISSUE THREAD

**Issue by bwaidelich** [neos/neos-ui#1772]
### Description

For inline editable properties on the Document node, the CKEditor is not displayed correctly.

### Steps to Reproduce

1. Add an inline-editable property to some *Document* NodeType configuration:

```yaml
'Neos.Neos:Document':
  'properties':
    'test':
      type: string
      ui:
        inlineEditable: true
```

2. Make sure the inline editable is rendered on the page
3. Focus the corresponding text in the Neos backend

#### Expected behavior

The Dimension Selector above the iFrame should be replaced by the CKEditor

#### Actual behavior

The CKEditor is not shown, instead the Dimension Selector stays visible:

![image](https://user-images.githubusercontent.com/307571/38806243-ea9dede0-4178-11e8-9e83-872d2b0ac0ca.png)

*Note:* The inline-editing itself works, but the CKEditor controls are not usable

### Affected Versions

Neos:

3.3.9

UI:

1.0.9

**Comment by dimaip** [comment]
Hey @bwaidelich! Can't really confirm that. Do you have a correct ContentEditableWrapping around your page?

**Comment by dimaip** [comment]
Oh wait, I just checked again and it seems to be the case, I think I know what's wrong...

**Comment by DrillSergeant** [comment]
Maybe duplicate of https://github.com/neos/neos-ui/issues/1711? I think this was solved in 1.0.10 or 1.0.11. In my installation (1.0.11) I can't reproduce it.

**Comment by dimaip** [comment]
Oh yes, I tested on an older installation as well...
